src/Dashboard/Dashboard.tsx

```diff
@@ -14,13 +14,12 @@ const Dashboard: React.SFC<DashboardProps> = (props) => {
 
 	return (
 		<section className='cocktail-OTD-wrapper'>
-			<p className='welcome-msg'>Welcome {props.username}!</p>
 			<p className='COTD-title'>Cocktail of the Day</p>
 			{error && <div>{error}</div>}
 			<section className='cocktail-OTD'>
 				<h3>{randomCocktail.strDrink}</h3>
 				<img className='COTD-img' src={`${randomCocktail.strDrinkThumb}`} alt={`${randomCocktail.strDrink}`} /> 
-				<Link to={`/${randomCocktail.idDrink}/details`}><button className='details-btn' aria-label='details-button'>Make Me</button></Link>
+				<Link to={`/cocktails/${randomCocktail.idDrink}`}><button className='details-btn' aria-label='details-button'>Make Me</button></Link>
```

Comment:
changed path routing to be more RESTful, per Khalids suggestion @Melizzo @asodemann18 @LeahWinters